testsuite/integration-arquillian/tests/base/pom.xml

```diff
@@ -44,6 +44,11 @@
     </properties>
     
     <dependencies>
+        <dependency>
```

Comment:
I agree with you. I also would like to remove this dependency. 

I think it might be sufficient to realize the following:

* On conducting tests under webauthn package, use ChromeDriver and set "--enable-web-authentication-testing-api" option.

It is better to do that using exsinging arquillian/drone system. However, I'm not sure how to do it.

Therefore, I've implement the test codes by other way that actually run for the moment. 

As you advised, I'll look into AbstractX509AuthenticationTest.

Reply:
This is definitely not a blocker for this PR IMO as I mentioned above.

We can create follow-up JIRA and someone from Keycloak team, who is more familiar with arquillian/drone can take a look into it. For now, the test is @Ignored anyway in the 1st stage of the PR, so I prefer to have something working in the keycloak master ASAP and then figure those details later :)

Reply:
I see. I'll work other tasks first.

Reply:
I've removed this dependency of Web Driver manager and use Arquillian Drone managing Chrome Driver. If you want to run the test, please do the following :
 
`mvn -f testsuite/integration-arquillian/tests/base/pom.xml test -Dtest=org.keycloak.testsuite.webauthn.WebAuthnRegisterAndLoginTest -Dbrowser=chrome  -DchromeArguments="--headless --enable-web-authentication-testing-api"`

I've looked into http://arquillian.org/arquillian-extension-drone/#chrome-options and integration-arquillian-tests-base/src/test/resources/arquillian.xml .

Reply:
Thanks, in my environment it unfortunately didn't help. Still the same behaviour. But thanks for this change anyway.